installer/win/Symphony-x64.aip

```diff
@@ -161,6 +161,7 @@
     <ROW Component="build.cmd" ComponentId="{2892AEA1-7773-46FE-8F4C-56E7EAC3BDC3}" Directory_="spawnrx_Dir" Attributes="0" KeyPath="build.cmd" Type="0"/>
     <ROW Component="cld.node" ComponentId="{2598E57C-15B4-4B21-9DDA-EBD760F45C79}" Directory_="Release_Dir" Attributes="256" KeyPath="cld.node" Type="0"/>
     <ROW Component="d3dcompiler_47.dll" ComponentId="{C7B87C02-3116-43A8-A70B-3592B70E6AC8}" Directory_="APPDIR" Attributes="256" KeyPath="d3dcompiler_47.dll"/>
+    <ROW Component="dictionary" ComponentId="{0337D363-6EC5-4283-98AC-0E2931979F53}" Directory_="library_Dir" Attributes="0" KeyPath="dictionary" Type="0"/>
```

Comment:
are we essentially just including a new dll here or is this something else?

Reply:
The changes here are for including the new file i.e `dictionary` file